_includes/api-docs.html

```diff
@@ -8,7 +8,17 @@
 {% assign versions = api.versions | sort | where_exp: "item", "item[1] == 1" | join: "" | split: "1" | reverse %}
 ### {{ api.name }}
 {{ api.desc }}
-#### DOCS {% for version_name in versions %} {% if api.name == "libucxx" %} **[{{ version_name }} ({{ site.data.releases[version_name].ucxx_version }})](/api/{{ api.path }}/{{ version_name }})** {% else %} **[{{ version_name }} ({{ site.data.releases[version_name].version }})](/api/{{ api.path }}/{{ version_name }})** {% endif %} {% unless forloop.last %}|{% endunless %} {% endfor %}
+#### DOCS {% for version_name in versions -%}
+    {%- if api.version-overrides -%}
+        **[{{ version_name }} ({{ api.version-overrides[version_name] }})](/api/{{ api.path }}/{{ version_name }})**
+    {%- elsif api.name == "libucxx" -%}
```

Comment:
Check this? elsif seems unusual.
```suggestion
    {%- elif api.name == "libucxx" -%}
```

Reply:
I was as surprised as you are to learn that in Liquid, it's literally spelled `elsif` 😅 

https://shopify.github.io/liquid/tags/control-flow/

Reply:
Ha no problem, I only just learned that recently through this work.